src/assets/source/scripts/main.js

```diff
@@ -74,6 +75,9 @@ require.config({
         underscore: {
             exports: '_'
         },
+        d3: {
+            exports: 'd3'
```

Comment:
Is this needed? I think `serlo_math_puzzle` uses `d3`, too, and didn't have this line